W3ChampionsStatisticService/PersonalSettings/ProfilePicture.cs

```diff
@@ -1,15 +1,17 @@
-﻿using W3C.Domain.CommonValueObjects;
+﻿using System;
+using W3C.Domain.CommonValueObjects;
 
 namespace W3ChampionsStatisticService.PersonalSettings
 {
     public class ProfilePicture
     {
         public static ProfilePicture Default()
         {
+            var random = new Random();
             return new ProfilePicture()
             {
-                Race = AvatarCategory.Total,
-                PictureId = 0
+                Race = AvatarCategory.Starter,
+                PictureId = random.Next(1,5),
```

Comment:
This will not generate the id 5, but there are 5 starter icons in the definition of the PR description. So the 5th would never be the default @Cepheid-UK

Reply:
I am also renaming the icon portraits to start at 1, instead of 0 as before.